api/src/main/resources/workbench-api.yaml

```diff
@@ -640,6 +640,47 @@ paths:
               schema:
                 $ref: '#/components/schemas/ErrorResponse'
       x-codegen-request-body-name: runtimesToDelete
+  /v1/admin/security/workspaces/{workspaceNamespace}/deleteRuntimes:
+    post:
+      tags:
+        - workspaceAdmin
+      summary: Delete specified (or all if no specified list) runtimes in given workspace
+        namespace.
+      description: An admin gated endpoint that deletes given runtimes in a given
+        workspace.
+      operationId: adminDeleteRuntimes
+      parameters:
+        - name: workspaceNamespace
+          in: path
+          description: The workspace's namespace.
+          required: true
+          schema:
+            type: string
+      requestBody:
+        description: A list of names of runtimes to delete. To delete all runtimes,
```

Comment:
I am unclear about the distinction here.

It seems like {} will delete all and [] will delete nothing.

Reply:
that's right.

It's also a bit surprising.  Maybe I should take this opportunity to improve the signature.

Reply:
hm, this is proving more involved than I expected.  I'll save that for a second PR.

Reply:
https://github.com/all-of-us/workbench/pull/8884